manifests/m/MicroSIP/MicroSIP/Lite/3.21.3/MicroSIP.MicroSIP.Lite.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: MicroSIP.MicroSIP.Lite
+PackageVersion: 3.21.3
+PackageLocale: en-US
+Publisher: MicroSIP

```

```suggestion
Publisher: www.microsip.org
```
This must match Control Panel.